SlicerLite/SlicerLiteLib/SlicerLiteModuleWidget.py

```diff
@@ -0,0 +1,170 @@
+import qt, slicer
+
+from .ButtonItemDelegate import DeleteButtonItemDelegate, DicomMetadataButtonItemDelegate
+from .DataLoader import DataLoader
+from .EventFilters import DragAndDropEventFilter
+from .UIUtils import UIUtils
+from .SlicerUtils import *
+from .ItemModel import ItemModel, Item
+
+
+class SlicerLiteModuleWidget(qt.QWidget):
+    def __init__(self, parent=None):
+        super(SlicerLiteModuleWidget, self).__init__(parent)
+        qt.QVBoxLayout(self)
+        self._lastOpenedDirectory = ""
+
+        self.data_loader = DataLoader()
+        self.item_table_model = ItemModel()
+        self.item_table_view = qt.QTableView()
+        self.item_table_model.dataChanged.connect(lambda _: self.item_table_view.viewport().update())
```

Comment:
You don't have to manually connect the update. Once you set the model to the table view, any model update will trigger the view update